contracts/swappa/PairBPool.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.8;
+
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "../interfaces/balancer/IBPool.sol";
+import "./ISwappaPairV1.sol";
+
+contract PairBPool is ISwappaPairV1 {
+	using SafeMath for uint;
+
+	function swap(
+		address input,
+		address output,
+		address to,
+		bytes calldata data
+	) external override {
+		address poolAddr = parseData(data);
+		uint inputAmount = ERC20(input).balanceOf(address(this));
+		require(
+			ERC20(input).approve(poolAddr, inputAmount),
+			"PairBPool: approve failed!");
+		(uint outputAmount,) = IBPool(poolAddr).swapExactAmountIn(
+			input,
+			inputAmount,
+			output,
+			0, // minAmountOut
+			~uint256(0) // maxPrice
```

Comment:
taking a short cut here, `minAmountOut` = 0 because we already have output check in swappa at the end,
same goes for `maxPrice` cus we always do our own output check, so no need for computing these parameters